src/Service.js

```diff
@@ -62,28 +72,35 @@ export default class Service {
     return slot;
   }
 
+  /**
+   * Removes the given slot from the service.
+   *
+   * @param {Slot} slot The slot to remove
+   * @private
+   */
   _removeSlot(slot) {
     const index = this._slots.indexOf(slot);
     if (index !== -1) {
       this._slots.splice(index, 1);
-      this._gt.removeSlot(slot);
-      const id = `${slot.getAdUnitPath()}_${index}`;
+      this._gpt._removeSlot(slot);
+      const id = `${slot.getAdUnitPath()}_${index + 1}`;
       delete this._slotIdMap[id];
     }
   }
 
   /**
-   * UNDOCUMENTED - Returns a map of ID's to slots.
+   * Returns a map of ID's to slots.
    *
-   * @returns {Object<string, Slot>} The map of ID's to slots.
+   * @experimental
+   * @returns {object} The map of ID's to slots.
```

Comment:
Should `object` not be `Object`? I see it capped here: http://usejsdoc.org/tags-typedef.html